astropy/time/core.py

```diff
@@ -30,7 +30,6 @@
                       TimeJD, TimeUnique, TimeAstropyTime, TimeDatetime)
 # Import TimeFromEpoch to avoid breaking code that followed the old example of
 # making a custom timescale in the documentation.
-from .formats import TimeFromEpoch
```

Comment:
@mhvk - was the original example ever released?  I wrote the original example that imported from `core` on June 24, 2015, and you made the refactor updated on Sept. 1, 2015.  I would guess that the original doc update in #3885 was ever released.